openshift/templates/c_haines.cronjob.yaml

```diff
@@ -0,0 +1,119 @@
+  - name: POSTGRES_READ_HOST
+    required: true
+  - name: POSTGRES_USER
+    required: true
+  - name: POSTGRES_DATABASE
+    required: true
+objects:
+  - kind: CronJob
+    apiVersion: batch/v1beta1
+    metadata:
+      name: ${JOB_NAME}
+      labels:
+        cronjob: ${JOB_NAME}
+    spec:
+      schedule: ${SCHEDULE}
+      concurrencyPolicy: "Replace"
+      jobTemplate:
+        metadata:
+          labels:
+            cronjob: ${JOB_NAME}
+        spec:
+          template:
+            spec:
+              containers:
+                - name: ${JOB_NAME}
+                  resources:
+                    limits:
+                      cpu: "0.5"
+                      memory: 2Gi
+                    requests:
+                      cpu: "0.5"
+                      memory: 1Gi
+                  image: ${IMAGE_REGISTRY}/${PROJ_TOOLS}/${IMAGE_NAME}:${IMAGE_TAG}
+                  imagePullPolicy: "Always"
+                  command: ["python3", "-m", "app.c_haines.worker"]
+                  env:
+                    - name: POSTGRES_READ_USER
+                      value: ${POSTGRES_USER}
+                    - name: POSTGRES_WRITE_USER
+                      value: ${POSTGRES_USER}
+                    - name: POSTGRES_PASSWORD
+                      valueFrom:
+                        secretKeyRef:
+                          name: ${GLOBAL_NAME}-global
+                          key: app-db-password
+                    - name: POSTGRES_WRITE_HOST
+                      value: ${POSTGRES_WRITE_HOST}
+                    - name: POSTGRES_READ_HOST
+                      value: ${POSTGRES_READ_HOST}
+                    - name: POSTGRES_PORT
+                      value: "5432"
+                    - name: POSTGRES_DATABASE
+                      value: ${POSTGRES_DATABASE}
+                    - name: MAX_AGE_C_HAINES
+                      valueFrom:
+                        configMapKeyRef:
+                          name: ${GLOBAL_NAME}-global
+                          key: env.max-age-c-haines
+                    - name: ROCKET_URL_POST_MESSAGE
+                      valueFrom:
+                        configMapKeyRef:
+                          name: ${GLOBAL_NAME}-global
+                          key: rocket.chat-url-post-message
+                    - name: ROCKET_CHANNEL
+                      valueFrom:
+                        configMapKeyRef:
+                          name: ${GLOBAL_NAME}-global
+                          key: rocket.chat-channel
+                    - name: ROCKET_USER_ID
+                      valueFrom:
+                        secretKeyRef:
+                          name: ${GLOBAL_NAME}-global
+                          key: rocket.chat-user-id-secret
+                    - name: ROCKET_AUTH_TOKEN
+                      valueFrom:
+                        secretKeyRef:
+                          name: ${GLOBAL_NAME}-global
+                          key: rocket.chat-auth-token-secret
```

Comment:
the Rocketchat-related config and secret variables haven't been configured in OCP4 yet - just a heads up in case you were planning on relying on these notifications right now

Reply:
@andrea-williams weird - I'm seeing it, and deployment of other cronjobs shouldn't be possible if it's not working. Am I missing something?

Reply:
Whoops. I was looking in TEST, where it hasn't been configured. Everywhere else it has been. Silly me.